index.js

```diff
@@ -29,7 +29,15 @@ function toJson(value, options) {
   lines = value.split('\n')
 
   if (options.comment) {
-    lines = lines.map(stripComments(options.comment))
+    var comments = options.comment
+      ? Array.isArray(options.comment)
+        ? options.comment
+        : [options.comment]
+      : []
```

Comment:
Could you move this to the top? L11 for example?

Reply:
And then you don’t need to wrap this in the `if (options.comment) { ... }` anymore, because comments is always an array that can be iterated by `comments.forEach(function (comment) { ... }`

Reply:
Sorry, I'm still new to programming, I'll try to execute what you asked for, I'm just out of time at the moment but i'll try.

Reply:
No problem! Let me know if you have questions, or if you’d like me to do it myself!

Reply:
I never write a test for example, but i know i can

Reply:
Then you should! Good luck! 🤗

Reply:
anything I can do to help?

Reply:
Sorry dude, i don't know write a test and sorry for the silly doubts, but since you set out to help...
My most difficult is "what exactly do i need to test?".
Also, I look at your test and I can't understand, how he will guarantee something, or how it will serve to prevent a future error.

Reply:
Here’s what I was looking for: https://github.com/wooorm/plain-text-data-to-json/commit/d5c7cb5510d61a888d53161c57be20ffd4796631.

This test makes sure your new added feature, supporting multiple comments, works.